resultsdbupdater/consumer.py

```diff
@@ -18,15 +22,20 @@ def debug_log_msg(self, msg):
         LOGGER.debug(str(msg))
 
     def consume(self, msg):
-        if 'headers' in msg:
-            if 'CI_TYPE' in msg['headers']:
-                if msg['headers']['CI_TYPE'] == 'ci-metricsdata':
-                    self.debug_log_msg(msg)
-                    return ci_metrics_post_to_resultsdb(msg)
-                elif msg['headers']['CI_TYPE'] == 'resultsdb':
-                    self.debug_log_msg(msg)
-                    return resultsdb_post_to_resultsdb(msg)
-            elif 'ci_type' in msg['headers']:
-                if msg['headers']['ci_type'] == 'ci-tps':
-                    self.debug_log_msg(msg)
-                    return tps_post_to_resultsdb(msg)
+        resultsdb_style_topics = [
+            '/topic/VirtualTopic.eng.platformci.covscan.result',
+            '/topic/VirtualTopic.qe.ci.jenkins'
```

Comment:
hmm, I bet this topic is definitely not resultsdb_style like.

Reply:
It is, but I do need to add additional filtering to make sure it is for the AMI test results.